spec/lib/guard/rspec_formatter_spec.rb

```diff
@@ -9,6 +9,12 @@ def rspec_summary_args(*args)
       [n.new(*args)]
     end
 
+    # Returns a Hash that throws an error if a key is accessed that isn't set.
+    def hash_double(attrs)
+      dbl = Hash.new{|hsh, key| raise "Missing key: #{key.inspect}." }
```

Comment:
Unused block argument - `hsh`. If it's necessary, use `_` or `_hsh` as an argument name to indicate that it won't be used.<br>Space missing to the left of {.<br>Space between { and | missing.